hibernate-core/src/main/java/org/hibernate/cache/spi/support/AbstractReadWriteAccess.java

```diff
@@ -198,13 +198,18 @@ protected void handleLockExpiry(SharedSessionContractImplementor session, Object
 	public void remove(SharedSessionContractImplementor session, Object key) {
 		if ( getStorageAccess().getFromCache( key, session ) instanceof SoftLock ) {
 			log.debugf( "Skipping #remove call in read-write access to maintain SoftLock : %s", key );
-			// don'tm do anything... we want the SoftLock to remain in place
+			// don't do anything... we want the SoftLock to remain in place
 		}
 		else {
 			super.remove( session, key );
 		}
 	}
 
+	@Override
+	public void removeAll(SharedSessionContractImplementor session) {
```

Comment:
Adding this no-op method here overrides the method in ``AbstractCachedDomainDataAccess`` (which calls ``clearCache``).

 This is invoked by the constructor in ``BulkOperationCleanupAction.EntityCleanup``, which is called during the transaction (so the no-op here avoids the cache being cleared prior to the transaction committing).